repository/kafka/docs/v0.1.1/configuration.md

```diff
@@ -0,0 +1,55 @@
+# Configuration 
+
+### Resources
+
+By default, KUDO Kafka resource configuration is set to be minimal optimal for the production usage. 
+But users can tune the requirements depending on high or low workload of the Kafka cluster.  
+
+##### Tuning the resources for the Kafka Cluster
+
+```
+kubectl kudo install kafka --instance=my-kafka-name \
+            -p ZOOKEEPER_URI=zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 \
+            -p ZOOKEEPER_PATH=/custom-path \
+            -p BROKER_CPUS=3000m
+            -p BROKER_COUNT=5
+            -p BROKER_MEM=4096m
+            -p DISK_SIZE=20Gi
+            -p MIN_INSYNC_REPLICAS=3
+            -p NUM_IO_THREADS=10
+            -p NUM_NETWORK_THREADS=5
+          
+```
+
+##### Storage
+
+By default, the Kafka operator will use the default storage class of the Kubernetes cluster. 
+
+To deploy the Kafka cluster using a storage class that isn't the default, you can use the parameter `STORAGE_CLASS`
+
+```
+kubectl kudo install kafka --instance=my-kafka-name -p STORAGE_CLASS=<STORAGE_CLASS_NAME>
+```
+
+##### Deploying an ephemeral Kafka cluster without persistence
+
+```
+kubectl kudo install kafka --instance=my-kafka-name -p PERSISTENT_STORAGE=false
+```
+
+Deploying without persistent storage isn't recommended for production usage. 
```

Comment:
does this mean that there is no external storage and the data is destroyed after the container is rescheduled or stopped? If so, do you think it makes sense to be more specific here?

Reply:
> does this mean that there is no external storage and the data is destroyed after the container is rescheduled or stopped?

right. operators have been using the word `ephemeral` deployment for `statefulsets` for this kind of configuration. We can be more specific and maybe later add a full section around storage.